frontend/src/routes/ClientScreens/ClientPlan.js

```diff
@@ -78,9 +69,8 @@ class ClientPlan extends Component {
   render() {
     const { useCanvas } = this.state;
     const BarSeries = useCanvas ? VerticalBarSeriesCanvas : VerticalBarSeries;
-    function Step1(props) {
-      var incomeval = 87000;
 
+    function EmergencySavings(props) {
```

Comment:
i just noticed that u make ur functions start with capital letters, is this a react thing? cuz i have always used camelCase for functions lol

Reply:
The way I understand how this works is that Emergency Savings in this example is treated like a component.  Check out line 367 in ClientPlan.js: <EmergencySavings currentStep={this.state.currentStep} />.  
I am just keeping the styles common between "component elements".  For example the button components created were also uppercase for example SecondaryButton.

Here is a link to how I implemented the multistep form.  It explains a little bit more how it works: https://css-tricks.com/the-magic-of-react-based-multi-step-forms/